Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.13.9"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
```

Comment:
Can we use `JSON3.jl` instead of `JSON.jl`? `JSON3` is more modern and maintained, we shouldn't introduce old packages.

Also, it would be nice to add version of json package to compat section.